config/default.yml

```diff
@@ -783,6 +783,11 @@ PrePush:
     description: 'Running RSpec test suite'
     required_executable: 'rspec'
 
+  Minitest:
+    enabled: false
+    description: 'Running Minitest test suite'
+    command: ['bundle', 'exec', 'rake', 'test']
```

Comment:
@sds This doesn't work for me. Here you're not invoking the tool either... you're telling `ruby` to "run a folder"...

```
$ bundle exec ruby -Ilib:test test
/home/deivid/.rubies/ruby-2.3.0/bin/ruby: Is a directory -- test (LoadError)
```